documentation/src/main/asciidoc/topics/ref_hotrod_data_format.adoc

```diff
@@ -1,16 +1,13 @@
-[id='hotrod_data_formats']
-= Configuring Hot Rod Client Data Formats
-By default, Hot Rod client operations use the configured marshaller when
-reading and writing from {brandname} servers for both keys and values.
+[id='hotrod-data-format_{context}']
+= Hot Rod DataFormat API
```

Comment:
I would put this before the "Data conversion on demand" section as this will be the more common use-case.